ansible/playbooks/tasks/cluster-bootstrap.yaml

```diff
@@ -243,7 +243,9 @@
     - cloud_platform_is_aws
     - not (use_sbd | bool)
   register: stonith_config_result
-  failed_when: "'ERROR' in stonith_config_result.stderr"
+  failed_when: >
+    (ansible_distribution_major_version == '12' and 'ERROR' in stonith_config_result.stderr and 'ERROR: warning' not in stonith_config_result.stderr) or
```

Comment:
To avoid a so complicate condition maybe we can:

1. has a task to compose the command
```
- name: Set corosync facts
  ansible.builtin.set_fact:
aws_stonith_cmd: >
    crm configure primitive rsc_aws_stonith stonith:external/ec2
    op start interval=0 timeout=180
    op stop interval=0 timeout=180
    op monitor interval=120 timeout=60
    meta target-role=Started
params tag={{ aws_stonith_tag}} pcmk_delay_max=15
```

2. one task to run it on 12sp5
```
- name: Configure AWS EC2 STONITH in 12sp5
  ansible.builtin.command: {{ aws_stonith_cmd }}
  when:
    - is_primary
    - cloud_platform_is_aws
    - not (use_sbd | bool)
    - ansible_distribution_major_version == '12' 
  register: stonith_config_result
  failed_when:
    - 'ERROR' in stonith_config_result.stderr
    - 'ERROR: warning' not in stonith_config_result.stderr
```

3. one task to run it on 15
```
- name: Configure AWS EC2 STONITH in 15
  ansible.builtin.command: {{ aws_stonith_cmd }}
  when:
    - is_primary
    - cloud_platform_is_aws
    - not (use_sbd | bool)
    - ansible_distribution_major_version != '12' 
  register: stonith_config_result
  failed_when:
    - 'ERROR' in stonith_config_result.stderr
```